### ISSUE TITLE
Translate message for version 2.5

### ISSUE THREAD

**Issue by dagargo** [dagargo/elektroid#89]
@snd1, @trebmuh and @xfgusta, Elektroid 2.5 is ready and I'm asking you once again to add a few new translations.

There are 2 use cases that required translations in this release:

1. The MicroBrute editor panel ported from https://github.com/dagargo/microdude (now archived). @trebmuh, French translations are already ported from your own translations there so you'll just need to take a look.
2. A user message (and action buttons related messages) to ask for file replacement if the file already exists.

TIA.

**Comment by trebmuh** [comment]
Done for the French translation. I only had to translate 3 strings.

**Comment by dagargo** [comment]
Merged.

Thanks, @trebmuh.

**Comment by snd1** [comment]
I tried my best ;-)

**Comment by dagargo** [comment]
Merged.

_Das ist super!_ Some messages are hard to translate and look better in plain English. But it looks great.

Thanks, @snd1.

**Comment by dagargo** [comment]
@xfgusta, I'd like to release version 2.5 and we depend on this issue. Do you think you could manage to add the 2.5 translations during the following week?

In case you can't, which is totally fine, there is no problem. We leave this open and carry on with the release.

Let us know about this. Thanks.

**Comment by xfgusta** [comment]
Hi @dagargo,

I apologize for the delay in updating the translation. I've been quite busy lately, and to be honest, I was feeling a bit lazy as well 😅

**Comment by dagargo** [comment]
No worries, @xfgusta. It's just on time. Thanks! :+1: 

Thanks to the translation team for taking the time to review these new messages.
